cmd/start.go

```diff
@@ -25,6 +28,17 @@ is already running or if no subnets have been deployed.`,
 }
 
 func startNetwork(cmd *cobra.Command, args []string) error {
+	sd := subnet.NewLocalSubnetDeployer(log, baseDir)
+
+	if err := sd.StartServer(); err != nil {
+		return err
+	}
+
+	avalancheGoBinPath, pluginDir, err := sd.GetBinPaths()
```

Comment:
It's kind of weird that `GetBinPaths` is hooked on the `LocalSubnetDeployer`. The paths are not really dependent on the deployer, but are general for the whole tool.

We can maybe change this in a future PR though.

Reply:
maybe I can change the name to SetupLocalEnv, it is really that previous function (setupLocalEnv) with extra checks